### PR TITLE
Use custom uri scheme for license links

### PR DESCRIPTION
**Pull request by xjuan**
Based on Martin workaround

https://phabricator.endlessm.com/T12960

### CODE REVIEW

File: js/app/widgets/eknWebview.js [hunk 1]
```diff
@@ -58,6 +59,7 @@ const EknWebview = new Knowledge.Class({
         'http',
         'https',
         'file',
+        'license'
```
Comment (ptomato) [inline js/app/widgets/eknWebview.js]: Nitpick: add trailing comma

File: js/app/articleHTMLRenderer.js [hunk 2]
```diff
@@ -296,6 +296,11 @@ function _to_link(uri, text) {
     return '<a class="eos-show-link" href="' + uri + '">' + Mustache.escape(text) + '</a>';
 }
 
+function _to_license_link(license) {
+    return _to_link ('license://' + GLib.uri_escape_string(license, null, false), Endless.get_license_display_name(license));
+}
+
```
Comment (ptomato) [inline js/app/articleHTMLRenderer.js]: Nitpick: remove extra blank

File: js/app/modules/layout/articleStack.js [hunk 3]
```diff
@@ -251,7 +251,7 @@ const ArticleStack = new Module.Class({
                 }
                 this._webview_tooltip_presenter.show_default_tooltip(tooltip, article_model.title);
             });
-        } else if (GLib.uri_parse_scheme(uri) === 'file' && uri.indexOf('/licenses/') > -1) {
+        } else if (GLib.uri_parse_scheme(uri) === 'license') {
             // If the uri has the "file://" scheme and it includes a segments for "licenses",
             // it corresponds to a license file, and we should display it as an external link.
```
Comment (xjuan) [inline js/app/modules/layout/articleStack.js]: We need to update the comment too. other than that CR pass!

**Comment by ptomato** [comment]
Fixed the comment.

**Pull request merged**